torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -1117,18 +1131,28 @@ void CudaCodeGen::call_raw(const std::vector<void*>& raw_args) {
       gpu_block_extents_v[i] = immediateAs<int64_t>(gpu_block_extents[i]);
       continue;
     }
+#ifdef TORCH_ENABLE_LLVM
+    gpu_block_extents_v[i] =
+        block_extents_compiled_[i].value<int64_t>(raw_args);
+#else
     ExprEval<SimpleIREvaluator> eval(
         ExprHandle(gpu_block_extents[i]), buffer_args);
     gpu_block_extents_v[i] = eval.value<int64_t>(raw_args);
+#endif
   }
   for (size_t i = 0; i < gpu_thread_extents.size(); i++) {
     if (gpu_thread_extents[i]->isConstant()) {
       gpu_thread_extents_v[i] = immediateAs<int64_t>(gpu_thread_extents[i]);
       continue;
     }
+#ifdef TORCH_ENABLE_LLVM
+    gpu_thread_extents_v[i] =
+        thread_extents_compiled_[i].value<int64_t>(raw_args);
+#else
     ExprEval<SimpleIREvaluator> eval(
         ExprHandle(gpu_thread_extents[i]), buffer_args);
     gpu_thread_extents_v[i] = eval.value<int64_t>(raw_args);
+#endif
```

Comment:
I meant here we don't need the ifdef if `thread_extents_compiled_[i].value` works even when there is no LLVM. But I think you're right about the `thread_extents_compiled_` - I thought that we also have something like `thread_extents_interpreted_` somewhere.

Reply:
Done. The `#ifdef` here is now gone. I had to update the header file to change the vector type depending on whether LLVM is present though.